hub/util/class_label.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+
+import numpy as np
+
+
+def convert_to_idx(samples, class_names: List[str]):
+    idxs = []
+    additions = []
+    for sample in samples:
+        if isinstance(sample, str):
+            for i in range(len(class_names)):
```

Comment:
this for loop for every sample can be slow, use a mapping of class_name to idx instead?